dev/sparktestsupport/modules.py

```diff
@@ -19,10 +19,28 @@
 import itertools
 import os
 import re
+import glob
+
+from sparktestsupport import SPARK_HOME
 
 all_modules = []
 
 
+def _discover_python_unittests(paths):
+    if not paths:
+        return set([])
+    tests = set([])
+    pyspark_path = os.path.join(SPARK_HOME, "python")
+    for path in paths:
+        # Discover the test*.py in every path
+        files = glob.glob(os.path.join(pyspark_path, path, "test*.py"))
```

Comment:
Do we intentionally include `pyspark/pandas/tests.data_type_ops.testing_utils`?

Reply:
Thanks for reminder, it should be test_*.py"